ci-operator/config/openshift/certman-operator/openshift-certman-operator-master.yaml

```diff
@@ -48,6 +48,23 @@ tests:
   commands: make validate
   container:
     from: src
+- as: leading-zero-check
+  postsubmit: true
+  steps:
+    test:
+    - as: check-commit-message
+      commands: |
+        COMMIT_HASH=$(git rev-parse --short HEAD)
+        if [[ $COMMIT_HASH =~ ^0 ]]; then
+            echo "Git commit hash starts with a leading zero. Please generate a new commit hash."
```

Comment:
I would suggest expanding this error message a bit to explain that we're doing this due to an OLM bug (bonus points for linking the bug) and that the fix is to add a dummy commit